src/js/utils/debounce.js

```diff
@@ -6,3 +6,5 @@ export const debounce = (cb, timer) => {
     timeout = setTimeout(() => cb.apply(context, args), timer);
   };
 };
+
+export const timer = ({ theme }) => theme.global.debounceTimer;
```

Comment:
Can we export `debounceDelay` instead of `timer` here? "timer" seems too generic for what it means. In other words, I should be able to read the import statements and know what they refer to without having to read the code in utils.js.